crew

```diff
@@ -843,8 +846,8 @@ def resolve_dependencies_and_build
   ensure
     #cleanup
     unless @opt_keep
-      FileUtils.rm_rf Dir.glob("#{CREW_BREW_DIR}/*")
-      FileUtils.mkdir_p "#{CREW_BREW_DIR}/dest" #this is a little ugly, feel free to find a better way
+      FileUtils.rm_rf Dir.glob('#{CREW_BREW_DIR}/*'), verbose: @fileutils_verbose
```

Comment:
`"#{CREW_BREW_DIR}/*"` should have double instead of single quotes to expand the constant.